docs/tools/IDE/updating_firmware.md

```diff
@@ -2,46 +2,50 @@
 
 The Online Compiler integrates with Pelion Device Management's remote firmware update services.
 
-<span class="tips">**Tip**: The fastest way to learn about firmware updates in the Online Compiler is to try the [Quick Connect guide](https://cloud.mbed.com/guides/connect-device-to-pelion) to create the original application, and the [Quick Update guide](https://cloud.mbed.com/guides/pelion-firmware-update) to perform a firmware update.</span>
+<!---Link to Pelion pages on Update--->
+<span class="tips">**Tip**: The fastest way to learn about firmware updates in the Online Compiler is to try the [Quick Connect guide](../guides/connect-device-to-pelion) to create the original application, and the [Quick Update guide](../guides/pelion-firmware-update) to perform a firmware update.</span>
 
 <span class="notes">You need a [Pelion Device Management](https://portal.mbedcloud.com/login) account to use the update workflow.</span>
 
+<!---
 ## Creating an application you can update
 
 To be able to receive updates, an application must have:
 
-- A **connect certificate** to access a Pelion Device Management account. When you run the application on your device, the bootstrap server will use this connect certificate to authenticate your application with your Pelion Device Management account.
+- A **connect certificate** to access a Pelion Device Management account. When you run the application on your device, the bootstrap server will use this connect certificate to authenticate your application with your Pelion Device Management account. <!---We don't use 'connect certificate' anywhere else. Seems it might come from the online compiler.---><!---
```

Comment:
What do we call it elsewhere? Anything?

Reply:
If connect certificate is from the compiler, then there's a wider discussion to be had

Reply:
We don't call them anything but a certificate.

Reply:
Isn't that confusing? We have quite a few.

Reply:
https://www.pelion.com/docs/device-management/current/provisioning-process/certificates-and-certificate-authorities.html

Reply:
@madchutney and/or @mark-edgeworth from the Mbed OS tools side.

Reply:
@bertfroeba is 'interested' in this too...

Reply:
This might be using a "developer" certificate. Which I am on a crusade against.
No certificate should ever be compiled into an application. It obfuscates it's purpose and makes it harder to transition into production when doing so.

Reply:
I've sent an email to y'all so we can get this sorted.